src/Microsoft.PowerShell.Commands.Management/commands/management/TestConnectionCommand.cs

```diff
@@ -133,19 +134,22 @@ public class TestConnectionCommand : PSCmdlet, IDisposable
         public int MaxHops { get; set; } = DefaultMaxHops;
 
         /// <summary>
-        /// Gets or sets the number of ping attempts.
-        /// The default (from Windows) is 4 times.
+        /// Gets or sets the number of test attempts.
+        /// The value is -1 if not set explicitly as a parameter
+        /// If not set explicitly, the value is set depending on type of test (ping = 4, tcp = 1)
         /// </summary>
         [Parameter(ParameterSetName = DefaultPingParameterSet)]
+        [Parameter(ParameterSetName = TcpPortParameterSet)]
         [ValidateRange(ValidateRangeKind.Positive)]
-        public int Count { get; set; } = 4;
+        public int Count { get; set; } = -1;
```

Comment:
Better to not have a default value here; we can test whether it's been provided via bound parameters.

Reply:
I've undone my changes, and moved this back to it's original setting 🙂